dali/pipeline/graph/op_graph2.h

```diff
@@ -61,8 +61,6 @@ struct OpNode {
   const std::string instance_name;
   /** The specification of an operator. */
   OpSpec spec;
-  /** The "device" - cpu, gpu or mixed. */
-  OpType op_type;
```

Comment:
Actually, this field should be used. It may be unused at present but if it's uninitialized, then it's an omission in the constructor and not a superfluous field.

Reply:
Can you bring it back with the changes to come?